extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -161,7 +161,7 @@ export function getElement(selector, el, selectionMethod) {
   } else if (selectionMethod == 'scope') {
     return el.parentElement.querySelector(selector);
   } else if (selector[0] == '#') {
-    return el.ownerDocument.getElementById(selector.slice(1));
+    return ampdoc.getElementById(selector.slice(1));
```

Comment:
A note for the related code. Five lines above (I can't comment there) we do `closest` selectors - we should restrict the result to be contained by `ampdoc`